test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -1860,7 +1877,7 @@ class XdsEnd2endTest : public ::testing::TestWithParam<TestType> {
   void SetNextResolution(
       const std::vector<int>& ports,
       grpc_core::FakeResolverResponseGenerator* response_generator = nullptr) {
-    if (GetParam().use_xds_resolver()) return;  // Not used with xds resolver.
+    if (!GetParam().use_fake_resolver()) return;  // Not used with xds resolver.
```

Comment:
Does this imply we will have a non-fake non-xds resolver being tested in this file?

Reply:
No.  The only two options are the fake resolver and the xds resolver.  The latter is the overwhemlingly common case, so I changed the default accordingly.